Minor changes (#108852)
Azure/azure-content: articles/openshift/howto-gpu-workloads.md

   ```bash
   CHANNEL=$(oc get packagemanifest gpu-operator-certified -n openshift-marketplace -o jsonpath='{.status.defaultChannel}')
   ```

> > **_NOTE:_** If your cluster was created without providing the pull secret, the cluster will not include samples or operators from Red Hat or from certified partners. This will result in the following error message: 
> > *Error from server (NotFound): packagemanifests.packages.operators.coreos.com "gpu-operator-certified" not found.* To add your Red Hat pull secret on an Azure Red Hat OpenShift cluster, [follow this guidance](howto-add-update-pull-secret.md).

Reviewer: ```suggestion
> [!NOTE] 
> If your cluster was created without providing the pull secret, the cluster won't include samples or operators from Red Hat or from certified partners. This will result in the following error message: 
> 
> *Error from server (NotFound): packagemanifests.packages.operators.coreos.com "gpu-operator-certified" not found.* 
>
> To add your Red Hat pull secret on an Azure Red Hat OpenShift cluster, [follow this guidance](howto-add-update-pull-secret.md).
```

> > **_NOTE:_** Be aware that if your cluster was created without providing the pull secret, the cluster will not include samples or operators from Red Hat or from certified partners, which  will cause an error message saying: 
> > *Error from server (NotFound): packagemanifests.packages.operators.coreos.com "gpu-operator-certified" not found.* To add your Red Hat pull secret on an Azure Red Hat OpenShift cluster, [follow this guidance](howto-add-update-pull-secret.md).

Reviewer 2: ```suggestion
> **_NOTE:_** If your cluster was created without providing the pull secret, the cluster will not include samples or operators from Red Hat or from certified partners. This will result in the following error message: 
> *Error from server (NotFound): packagemanifests.packages.operators.coreos.com "gpu-operator-certified" not found.* To add your Red Hat pull secret on an Azure Red Hat OpenShift cluster, [follow this guidance](howto-add-update-pull-secret.md).
```
